Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM node:lts-alpine
+
+RUN \
+  apk update && \
+  apk add --no-cache \
+    python3 \
+    && \
+  cp /usr/bin/python3 /usr/bin/python && \
+  pip3 install \
+    pyspf \
+    dnspython \
+    dkimpy
+
+WORKDIR /app
+
+COPY package*.json ./
+COPY yarn.lock yarn.lock
+
+RUN npm i -g pm2 && npx yarn --prod
```

Comment:
- install global packages before copying over package.json (move this statement after pip install)
- yarn is bundled in this image, no need to use npx.